### ISSUE TITLE
By default it works but it doesn't choose the good vim instance

### ISSUE THREAD

**Issue by RoXuS** [artemave/tmux_super_fingers#19]
Hey, thx for this plugin!

Not sure if it is an issue but by default the plugin opens the file in other window (where an instance of nvim is present) instead of the current window where I have an existing pane with nvim too. 

Is it the expected behavior?

**Comment by artemave** [comment]
Yeah, that's expected. It's just picking the first nvim instance. 

Perhaps it could prefer the one in the current window, if there is one.

**Comment by RoXuS** [comment]
or maybe an option to pick the first nvim instance OR pick the closest (in current window then in closest window)?

**Comment by artemave** [comment]
By "first" I really meant unspecified. It was never designed to work with multiple vim instances. So I don't believe "first" has any value.

**Comment by RoXuS** [comment]
oh ok and this feature is possible, can we selected the "closest" vim instance?
